ui/src/tool-panel/location-track/location-track-infobox.tsx

```diff
@@ -367,6 +395,27 @@ const LocationTrackInfobox: React.FC<LocationTrackInfoboxProps> = ({
                                         </InfoboxButtons>
                                     </React.Fragment>
                                 )}
+                                <InfoboxButtons>
+                                    {!splittingState && (
+                                        <Button
+                                            variant={ButtonVariant.SECONDARY}
+                                            size={ButtonSize.SMALL}
+                                            onClick={() => {
+                                                if (
```

Comment:
Tämä menee selvästi liian syväksi kun ei tahdo mikään pysyä samalla rivillä. Simppeli siistiminen olisi tehdä tuo toiminnon funkkari ylempänä muuttujaan. Toisaalta myös näitä infobox-osioita kannattaisi splittailla omiin tiedostoihinsa tai vähintään erillisiin luontifunkkareihin, koska tämä on myös ylipitkä tiedosto ja järjettömän pitkä funktio.